extension/controller/experiments/TranslationBar/api.js

```diff
@@ -65,13 +65,18 @@ const translationNotificationManagers = new Map();
 
     getAPI(context) {
```

Comment:
Yes, this API is only called by the background script and there are no plans to change that since there's no other script or page that should trigger the translation bar (that would actually break the architecture).

I'd prefer to follow this up later if ever needed, since changes to this API and the underlying UI proved to be a bit brittle throughout the project, so I'm being overly cautious and would prefer to keep the changes minimal in order to fix the cache issue for now.

Reply:
Sure, as I wrote in my comment, at the moment it is fine to defer it.

I mainly wanted to confirm if that would become an issue if more pages are using the same experimental API now that I was already looking this part closely, and capture STR and possible approach to fix that in a comment that we can refer to in a followup issue.